doc/async_worker.md

```diff
@@ -160,31 +159,30 @@ information exposed by the async_hooks API.
 - `[in] resource`: Object associated with the asynchronous operation that
 will be passed to possible async_hooks.
 
-Returns an AsyncWork instance which can later be queued for execution by calling
+Returns a `Napi::AsyncWork` instance which can later be queued for execution by calling
 `Queue`.
 
 ### Constructor
 
-Creates a new `AsyncWorker`.
+Creates a new `Napi::AsyncWorker`.
 
 ```cpp
-explicit AsyncWorker(const Object& receiver, const Function& callback);
+explicit Napi::AsyncWorker(const Object& receiver, const Function& callback);
 ```
 
 - `[in] receiver`: The `this` object passed to the called function.
 - `[in] callback`: The function which will be called when an asynchronous
 operations ends. The given function is called from the main event loop thread.
 
-Returns an AsyncWork instance which can later be queued for execution by calling
+Returns a `Napi::AsyncWork` instance which can later be queued for execution by calling
 `Queue`.
 
-
 ### Constructor
 
-Creates a new `AsyncWorker`.
+Creates a new `Napi::AsyncWorker`.
 
 ```cpp
-explicit AsyncWorker(const Object& receiver, const Function& callback,const char* resource_name);
+explicit Napi::AsyncWorker(const Object& receiver, const Function& callback,const char* resource_name);
```

Comment:
nit: The types of the formal parameters should probably be fully namespaced.

Reply:
Done